frontend/src/app/components/MobileDialog/index.js

```diff
@@ -82,12 +82,23 @@ export default class MobileDialog extends React.Component {
 
     const headerImg = ios_url ? (<a href={ios_url} target="_blank"><img className="mobile-header-img" src="/static/images/ios-light.svg"/></a>) : (<a href={android_url} target="_blank"><img className="mobile-header-img" src="/static/images/google-play.png"/></a>);
 
+    const learnMoreLink = "https://www.mozilla.org/en-US/privacy/websites/#campaigns";

```

Comment:
nit: drop the "/en-US/" in the URL and let mozilla.org serve the "best" user locale.